x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/pipelines/inference_pipeline_card.tsx

```diff
@@ -222,6 +222,11 @@ export const InferencePipelineCard: React.FC<InferencePipeline> = (pipeline) =>
                 </EuiFlexGroup>
               </EuiFlexItem>
             )}
+            {pipeline.sourceFields && (
```

Comment:
should this be `pipeline.sourceFields && pipeline.sourceFields.length &&` to ensure we don't render this if the sourceFields is an empty list?

Reply:
Ok, added the condition.

Reply:
sorry it actually needs to be `!!pipeline.sourceFields.length` or `pipeline.sourceFields.length > 0` i forgot it will evaluate to “0” otherwise

Reply:
Yep, I used `pipeline.sourceFields.length > 0` 🙂